src/pages/rama-issue/issue.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import "./issue.css";
+import { useNavigate } from 'react-router-dom';
+import Pic from "./Group2.jpg";
+
+const Issue = () => {
+  const navigate2 = useNavigate();
+  return (
+    <div>
+    <div className="text">
+      <img alt="/" src={Pic} className="image1"/>
```

Comment:
This is a bad alt value. It should describe the image for accessibility or in case the image didn't render